roles/wildfly_prospero/defaults/main.yml

```diff
@@ -1,10 +1,10 @@
 ---
-wildfly_prospero_version: 1.1.0.Final
+wildfly_prospero_version: 1.2.0.Final
 wildfly_version: 29.0.0.Final
 wildfly_prospero_install_dir: /opt/prospero
 wildfly_prospero_profile_name: wildfly
 ldfly_prospero_scenario_enable: True
 wildfly_prospero_home: "/opt/prospero/prospero-{{ wildfly_prospero_version }}"
 wildfly_install_workdir: '/opt/wildfly/'
-wildfly_prospero_download_url: "https://repo1.maven.org/maven2/org/wildfly/prospero/prospero-build/{{ wildfly_prospero_version }}/prospero-build-{{ wildfly_prospero_version }}.tar.gz"
-wildfly_prospero_manifest_url: "https://raw.githubusercontent.com/wildfly-extras/prospero/{{ wildfly_prospero_version }}/examples/wildfly-27.0.0.Alpha2-manifest.yaml"
```

Comment:
@spyrkob Do we need to update this value regularly?

Reply:
We should only need to update the `wildfly_prospero_version` and `wildfly_version`. Hopefully we won't need to update prospero_version too often - unless there are some breaking changes in Galleon/Wildfly.

Reply:
I think it would be possible to remove the prospero version and always download the latest available version of prospero, using github API. I can take a look at it if that's something we want.